node/status_node_services.go

```diff
@@ -56,8 +56,6 @@ var (
 	// ErrRPCClientUnavailable is returned if an RPC client can't be retrieved.
 	// This is a normal situation when a node is stopped.
 	ErrRPCClientUnavailable = errors.New("JSON-RPC client is unavailable")
-	// OpenseaKeyFromEnv passed from on env var during build time
-	OpenseaKeyFromEnv string
```

Comment:
Searched for that symbol in `status-go`, `status-desktop` and `status-mobile`, found it nowhere else. Mobile seems to have moved to using the same approach we currently use in desktop [here](https://github.com/status-im/status-mobile/pull/13761).
However, this change will require changes in both `status-dektop` and `status-mobile` (since the `WalletConfig` member changed name and is now a map instead of a string). I could add some changes to make it backwards-compatible if the `status-mobile` people consider it better.

Reply:
I was reading some old information from when the testnet endpoints needed an API key. That's no longer the case, so the solution is much simpler.
No longer touching this part of the coide.